### ISSUE TITLE
trailing space is stripped from CDATA

### ISSUE THREAD

**Issue by esnosy** [martinblech/xmltodict#338]
for example this xml
```xml
<parent>
        <element><![CDATA[data    ]]></element>  
</parent>
```

let's parse it
```python
import xmltodict

xml = """
<parent>
        <element><![CDATA[data    ]]></element>
</parent>
"""
parsed_xml = xmltodict.parse(xml)
print(repr(parsed_xml['parent']['element']))
```

result:
`'data'`

expected result:
`'data    '`

untangle library is able to correctly parse it: https://pypi.org/project/untangle/
```python
import untangle

obj = untangle.parse(xml)
print(repr(obj.parent.element.cdata))
```
result
`'data    '`

**Comment by ibrahelsheikh** [comment]
yes i have same problem

**Comment by afbwilliam** [comment]
So, I encountered the same problem.  The solution is to pass "strip_whitespace=False" as an optional argument to xmltodict.parse().  So, for the above example, this should do the trick:

`import xmltodict

xml = """
<parent>
        <element><![CDATA[data    ]]></element>
</parent>
"""
parsed_xml = xmltodict.parse(xml, strip_whitespace=False)
print(repr(parsed_xml['parent']['element']))`

I discovered this after turning on debugging mode and stepping through the code.  It would be nice if xmltodict's user documentation was more robust, so users don't have dig through the code to investigate this in the first place.
